cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/azure/azure-mc-common/azure-orchestrator/AzureOrchestrator.service.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
```

Comment:
minor nitpick: please add comment separators '// React imports' '// App imports'

Reply:
Done. Will notice this in the future